.github/workflows/sbom-reachables.yaml

```diff
@@ -0,0 +1,31 @@
+name: SBOM Reachables
+on:
+  pull_request:
```

Comment:
Wouldn't it make more sense to have this done upon pushes to `master`?

Reply:
Using the same config from another workflow. So both master and release branches with some ignores.